Jenkinsfile

```diff
@@ -493,7 +493,7 @@ EOL
                 stage('Package and upload helm charts'){
                     steps {
                         script {
-                            docker.image("legion/legion-docker-agent:${Globals.buildVersion}").inside("-v /var/run/docker.sock:/var/run/docker.sock --net host") {
+                            docker.image("legion/legion-docker-agent:${Globals.buildVersion}").inside("-v /var/run/docker.sock:/var/run/docker.sock -u root") {
```

Comment:
Why did you add root user?
I think it a reason for the failed job https://cc.epm.kharlamov.biz/jenkins/job/Build_Legion_Artifacts/1655/